code/datums/ammo/xeno.dm

```diff
@@ -386,7 +386,7 @@
 	target.overlays += tail_image
 
 	new /datum/effects/xeno_slow(target, fired_proj.firer, ttl = 0.5 SECONDS)
-	target.apply_effect(0.5, STUN)
+	target.apply_effect(0.5, TRAIT_IMMOBILIZED)
```

Comment:
You don't apply traits with `apply_effect()` I suggest having a look at how `apply_effect()` works and which effects you can apply.

Reply:
oh fuck

Reply:
#5551